tests/memory_tests/src/memory_tests_helper/memory_counter.cpp

```diff
@@ -0,0 +1,69 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "memory_tests_helper/memory_counter.h"
+#include <fstream>
+#include <memory>
+#include <string>
+#include <cstring>
+#include <vector>
+
+#include "statistics_writer.h"
+
+namespace MemoryTest {
+#ifdef _WIN32
+size_t getVmSizeInKB() {
```

Comment:
Update current file to the latest version of file [stress_tests/common/utils.cpp](https://github.com/openvinotoolkit/openvino/blob/master/tests/stress_tests/common/utils.cpp)